core/lib/tasks/core.rake

```diff
@@ -177,4 +177,13 @@ use rake db:load_file[/absolute/path/to/sample/filename.rb]}
   task associate_orders_with_store: :environment do |_t, _args|
     Spree::Order.where(store_id: nil).update_all(store_id: Spree::Store.default.id)
   end
+
+  desc 'Ensure all Order has currency present after upgrading to Spree 3.7'
+  task ensure_order_currency_presence: :environment do |_t, _args|
+    Spree::Order.where(currency: nil).find_in_batches do |orders|
+      orders.each do |order|
+        order.update!(currency: order.store.default_currency || Spree::Config[:currency])
```

Comment:
@VygovskySergey maybe it will be safer to use `update_column` ?

Reply:
I thought about it, and decide that update will call `validation`, and if somebody doesn't set `store` to order, it will be set in `before_velidation callback`

Reply:
@VygovskySergey got it!